graylog2-web-interface/src/views/components/contexts/WidgetFocusProvider.test.tsx

```diff
@@ -80,4 +82,18 @@ describe('WidgetFocusProvider', () => {
     renderSUT();
     await screen.findByText('clack');
   });
+
+  it('should not set focused widget from url if the widget does not exist', async () => {
+    asMock(useStore).mockReturnValue({
```

Comment:
I think we should mock `WidgetStore` instead of `useStore` here. Imagine the following:

 - Instead of `useStore`, the SUT uses something different.
 - The SUT uses a second store with `useStore`.

In both cases the test would break, although the component/functionality works fine.